test/test_no_ssl.py

```diff
@@ -71,7 +71,7 @@ def setUp(self):
         sys.meta_path.insert(0, ssl_blocker)
 
     def tearDown(self):
-        assert sys.meta_path.pop(0) == ssl_blocker
+        sys.meta_path.remove(ssl_blocker)
```

Comment:
If we always insert it, why do we need to switch this to "remove"? Does the gcloud sdk install stuff on the `meta_path`?